source/MaterialXTest/ShaderGen.cpp

```diff
@@ -2354,21 +2464,30 @@ TEST_CASE("Osl Output Types", "[shadergen]")
         REQUIRE(shader != nullptr);
         REQUIRE(shader->getSourceCode().length() > 0);
         // Write out to file for inspection
-        // TODO: Use validation in MaterialXView library
         std::ofstream file;
-        file.open(exampleName + "_vector2.osl");
+        std::string fileName(exampleName + "_vector2.osl");
+        file.open(fileName);
         file << shader->getSourceCode();
         file.close();
 
+        // TODO: Use validation in MaterialXView library
+        std::string errorResult;
+        validateOSL(fileName, errorResult);
+        //REQUIRE(errorResult.size() == 0);
```

Comment:
Some of the tests have been disabled like this. Are these compilations failing now? Maybe add a TODO to fix the errors and re-enable the tests later.

Reply:
Actually, I think the majority of the tests fail for now. I'll leave a general TODO.